[ECO-1866] Assorted docs (#127)
econia-labs/emojicoin-dot-fun: doc/doc-site/docs/resources/faq.md

---
slug: /faq
title: 💬 Faq
hide_title: False
description: Documentation for emojicoin dot fun
---
Please note this is not financial advice and nothing the emojicoin monkey says should be taken seriously. He just likes to listen to himself talk and has caught the emojicoin bug.
![monkey](./monkey.png "monkey")

### Q: What is the cost to launch an Emojicoin? 
A:  1 APT — About the cost of a bushel of bananas

### Q: What is the market cap requirement for an Emojicoin to leave the bonding curve to the AMM?
A:  45,000 APT Market Cap

### Q: Assuming nobody sells their tokens, how much buy pressure is required for the Emojicoin reach a 45,000 APT market cap?
A:  10,000 APT

### Q: If an Emojicoin market is already launched, can I launch a second one of the same Emoji?
A:  No, once a particular Emojicoin trading pair is launched, you cannot launch another. You should potentially explore launching Emojicoin combinations instead. Get creative, don't be lame. 

### Q: Are emojicoins the future? 

> A: Emojis are a universal language, transcending borders and languages. 🌍 With the rise of decentralized finance and the integration of emojicoins, I believe will see a global shift where emojicoins become the standard for transactions. So get ready to live, laugh, and love with emojicoins! They're not just a trend; they're a revolution. Embrace the future, where every transaction is a smile, a laugh, and a story. Emojicoins are here to stay, and they're going to change the world one emoji at a time. 🌈🚀💖

Reviewer: ```suggestion
A: Emojis are a universal language, transcending borders and languages. 🌍 With the rise of decentralized finance and the integration of emojicoins, we believe we could witness a global shift where emojicoins become the standard for transactions. So get ready to live, laugh, and love with emojicoins! They're not just a trend; they're a revolution. Embrace the future, where every transaction is a smile, a laugh, and a story. Emojicoins are here to stay, and they're going to change the world one emoji at a time. 🌈🚀💖
```

Should be third person here, right?
